query-engine/connector-test-kit/src/test/scala/writes/uniquesAndNodeSelectors/relation_uniques/CompoundUniqueRelationFieldSpec.scala

```diff
@@ -14,245 +14,6 @@ import util._
-         |  parent Parent
-         |}
-       """
-    }
-    database.setup(project)
-
-    val res1 = server.query(
-      """
-        |mutation {
-        |  createParent(
-        |    data: {
-        |      id: 1
-        |      p: "Parent1"
-        |      child: {
-        |        create: {
-        |          id: 1,
-        |          c: "Child1"
-        |        }
-        |      }
-        |    }
-        |  ) {
-        |    id
-        |    p
-        |    child{
-        |       id
-        |       c
-        |    }
-        |  }
-        |}
-      """,
-      project
-    )
-
-    res1.toString() should be("{\"data\":{\"createParent\":{\"id\":1,\"p\":\"Parent1\",\"child\":{\"id\":1,\"c\":\"Child1\"}}}}")
-
-    val res2 = server.query(
-      """
-        |mutation {
-        |  updateParent(
-        |    where: {
-        |      child_id_p: {
-        |        child_id: 1,
-        |        p: "Parent1"
-        |      }
-        |    }
-        |    data: { p: { set: "UpdatedParent" }}
-        |  ) {
-        |    p
-        |  }
-        |}
-      """,
-      project
-    )
-
-    res2.toString() should be("{\"data\":{\"updateParent\":{\"p\":\"UpdatedParent\"}}}")
-
-    val res3 = server.query(
-      """
-        |mutation {
-        |  updateChild(where: { id: 1 } data: { parent: { update: { p: { set: "UpdatedFromChild" }}}}) {
-        |    parent { p }
-        |  }
-        |}
-      """,
-      project
-    )
-
-    res3.toString() should be("{\"data\":{\"updateChild\":{\"parent\":{\"p\":\"UpdatedFromChild\"}}}}")
-
-    val res4 = server.query(
-      """
-        |mutation {
-        |  upsertParent(
-        |    where: { child_id_p: { child_id: 2, p: "Parent2" } }
-        |    update: { p: { set: "Doesn't matter" }}
-        |    create: { id: 2, p: "Parent2", child: { create: { id: 2, c: "Child2" } } }
-        |  ) {
-        |    id
-        |    child {
-        |      id
-        |    }
-        |  }
-        |}
-        |
-      """,
-      project
-    )
-
-    res4.toString() should be("{\"data\":{\"upsertParent\":{\"id\":2,\"child\":{\"id\":2}}}}")
-  }
-
-  // Mutations in this test:
-  //  create         | root   | checked
-  //  update         | root   | checked
-  //  delete         | root   | not possible (1!:1!)
-  //  upsert         | root   | checked
-  //  updateMany     | root   | unnecessary
-  //  deleteMany     | root   | unnecessary
-  //  nested create  | create | checked
-  //  nested update  | update | checked
-  //  nested connect | -      | not possible (1!:1!)
-  //  nested disconn | -      | not possible (1!:1!)
-  //  nested delete  | -      | not possible (1!:1!)
-  //  nested set     | -      | not possible (1!:1!)
-  //  nested upsert  | -      | not possible (1!:1!)
-  //  nested deleteM | -      | not possible (1!:1!)
-  //  nested updateM | -      | not possible (1!:1!)
-  "Using a compound unique that includes a 1!:1! multi-field relation" should "work" in {
```

Comment:
We should probably bring this one back as it is not about 1:1 relations.

Reply:
There are other test cases in that file that test the compound unique behavior but with 1:1!, so I think its fine that this one is gone.